src/components/checkbox/ms-checkbox.ts

```diff
@@ -0,0 +1,46 @@
+import { LitElement, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+import { msCheckboxStyles } from "./ms-checkbox.styles";
+
+@customElement('ms-checkbox')
+export class MSCheckbox extends LitElement {
+    static styles = [msCheckboxStyles]
+
+    @property({ type: String })
+    title = '';
+
+    @property({ type: String })
+    description = '';
+
+    @property({ type: Boolean })
+    checked = false;
+
+    @property({ type: Boolean })
+    disabled = false;
+
+    render() {
+        return html`
```

Comment:
```suggestion
        return  this.renderWithStyles(html`
```
It should be wrapped with this method to override customizations

Reply:
Done!